src/list-data/request-action-mapping.ts

```diff
@@ -76,12 +76,12 @@ export const XproMapping: MappingI = {
 }
 
 export const ColinRequestActions = [
-  NrRequestActionCodes.AMALGAMATE,
   NrRequestActionCodes.CHANGE_NAME,
   NrRequestActionCodes.CONVERSION,
   NrRequestActionCodes.RESTORE
 ]
 
+// *** TODO: should these be from FF instead?
 export const ColinRequestTypes = [
   EntityType.BC,
   EntityType.CC,
```

Comment:
This is used in companyRadioBtnApplicable() and I think we should use a FF instead.

Reply:
Well, someone working on Change Name / Restoration / Alteration will need to handle this (ie, for which entity types do we do a business lookup?).

cc: @JazzarKarim @chenhongjing @ketaki-deodhar

Reply:
Or maybe this isn't needed at all to decide whether to look up a business?

However, the store getter that uses this is used to determine whether to show numbered/named options...

Reply:
How about if I tackle this?

Reply:
Sev, here's how I'm doing it right now. I created a new getter called `showRestoreNowButton`. This getter returns a boolean whether we can restore by using the FF.

I'm using this getter next to `showColinButton`. So, `showColinButton || showRestoreNowButton`.

I didn't need to touch the ColinRequestTypes as of yet. 

I was thinking for the other types; Alteration for example, we would do the same.

Reply:
OK but this is different -- it's whether to show the named/numbered radio buttons or not.

Reply:
Using FF might be a good idea here.

Reply:
Wait, sorry, you're right.

I added an `|| business` to the template encapsulating the `companyRadioBtnApplicable`. I created a watcher to watch when the business object changes and I'm doing the work there in an ifRestoration block.

However, you're right. This list might have to change based on which flow we're in? Currently it works for restorations since this is what I need but it might be different for other flows.

Reply:
I'm not doing any more work on this. I think the existing code is OK (but #664 makes is clearer).

Reply:
Sounds good 👍